arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json

```diff
@@ -0,0 +1,14079 @@
+        "operationId": "ReplicationProtectedItems_PlannedFailover",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "Unique fabric name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "protectionContainerName",
+            "in": "path",
+            "description": "Protection container name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "replicatedProtectedItemName",
+            "in": "path",
+            "description": "Replication protected item name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "failoverInput",
+            "in": "body",
+            "description": "Disable protection input.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PlannedFailoverInput"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ReplicationProtectedItem"
+            }
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "deprecated": false,
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Execute planned failover": {
+            "$ref": "../examples/ReplicationProtectedItems_PlannedFailover.json"
+          }
+        }
+      }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/replicationProtectionContainers/{protectionContainerName}/replicationProtectedItems/{replicatedProtectedItemName}/remove": {
+      "post": {
+        "tags": [
+          "ReplicationProtectedItems"
+        ],
+        "summary": "Disables protection.",
+        "description": "The operation to disable replication on a replication protected item. This will also\r\n            remove the item.",
+        "operationId": "ReplicationProtectedItems_Delete",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/para
```

Comment:
As we had discussed in our e-mail discussions, it was not possible for us to take any change at the service end. We had decided take up any such change whenever we come up with new version in the service. We had by-off from your side on this.